app.py

```diff
@@ -1,82 +1,97 @@
 import requests
 import json
-from flask import request, Flask , session , redirect 
+from flask import request, Flask, session, redirect
 
 CLIENT_ID = ""          # wprowadź Client_ID aplikacji
-CLIENT_SECRET = ""      # wprowadź Client_Secret aplikacji    
+CLIENT_SECRET = ""      # wprowadź Client_Secret aplikacji
 TOKEN_URL = "https://accounts.spotify.com/api/token"
 AUTH_URL = "https://accounts.spotify.com/authorize?"
 REDIRECT_URI = "http://localhost:5000/authorize"
 SCOPE = 'user-read-private user-read-email user-read-currently-playing user-modify-playback-state'
-AUTH_LINK = AUTH_URL+  'response_type=code&client_id=' + CLIENT_ID + '&redirect_uri=' + REDIRECT_URI + '&scope='+ SCOPE
+AUTH_LINK = AUTH_URL + 'response_type=code&client_id=' + \
+CLIENT_ID + '&redirect_uri=' + REDIRECT_URI + '&scope=' + SCOPE
 
 app = Flask(__name__)
 
 app.secret_key = 'SOMETHING-RANDOM'
 app.config['SESSION_COOKIE_NAME'] = 'spotify-login-session'
 
+
 @app.route('/')
 def login():
     return redirect(AUTH_LINK)
 
+
 @app.route('/authorize')
 def authorize():
     code = request.args.get('code')
     access_token = get_access_token(code)
     session['access_token'] = access_token
     return redirect('/playlist')
 
+
 @app.route('/playlist')
 def playlist():
     if not 'access_token' in session:
         return redirect('/')
     access_token = session['access_token']
-    recomendation_music = get_Recommendations(access_token)
+    recommendation_music = get_Recommendations(access_token)
     for next_music in range(50):
-        get_music_queue(access_token,recomendation_music[next_music]['id'])
-    return 'done'
+        get_music_queue(access_token, recommendation_music[next_music]['id'])
+    return 'Done!!!!'
+
 
 def get_Recommendations(token):
     try:
-        actual_song = get_actual_play_music(token)
-        seed_artists = '?seed_artists=' +actual_song['artists'][0]['id']
-        seed_tracks = '&seed_tracks=' + actual_song['id']
-        url= "https://api.spotify.com/v1/recommendations" +seed_artists + seed_tracks +'&limit=50'
-        headers = {'Authorization': 'Bearer '+token , 'content_type': 'application/json'}
-        response = requests.get(url , headers=headers)
+        current_song = get_current_play_music(token)

```

Comment:
This is utterly a mess.

Reply:
Split this logic into few functions, separate URL, add headers management via router - don't add it in every function.